root.tf

```diff
@@ -673,6 +673,32 @@ module "bastion_role" {
   policy_attachments = {}
 }
 
+module "update_waf_and_security_groups_policy" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "UpdateWAFAndSecurityGroupsPolicy${title(local.environment)}"
+  policy_string = templatefile("${path.module}/templates/iam_policy/update_waf_and_security_groups_policy.json.tpl", { rule_group_arn = module.waf.rule_group_arn, ip_set_arn = module.waf.ip_set_arn, account_id = data.aws_caller_identity.current.account_id, security_group_id = module.frontend.alb_security_group_id })
+}
+
+module "update_waf_and_security_groups_role" {
+  source             = "./tdr-terraform-modules/iam_role"
+  assume_role_policy = templatefile("${path.module}/templates/iam_role/github_assume_role.json.tpl", { account_id = data.aws_caller_identity.current.account_id, repo_name = "tdr-e2e-tests" })
+  common_tags        = local.common_tags
+  name               = "UpdateWAFAndSecurityGroupsRole${title(local.environment)}"
+  policy_attachments = {
+    update_policy = module.update_waf_and_security_groups_policy.policy_arn
+  }
+}
+
+module "update_ecs_role" {
```

Comment:
Could the module be renamed slightly to make it clearer what this is: `github_update_ecs_role`